### PR TITLE
Link to code of conduct

### LINKED ISSUE #5154: [Discussion] Add a Code of Conduct for 1.0 release
With Deno fast approaching 1.0, it's time for us to finally have a Code of Conduct so as to set a baseline standard for expected behavior in the community. This will help both create a more inclusive community and set guidelines on how to act when a violation does occur.

Similar to #79, and #3954

Also PR #3956

### PR DESCRIPTION
**Pull request by ry**
Fixes #5154

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+
 - Please don't make [the benchmarks](https://deno.land/benchmarks.html) worse.
+
 - Ask for help in the [community chat room](https://discord.gg/TGMHGv6).
+
 - If you are going to work on an issue, mention so in the issue comments
   _before_ you start working on the issue.
 
+- Please be professional in the forums. Don't know what professional means?
+  [https://www.rust-lang.org/policies/code-of-conduct](Read Rust's code of
```
Comment (SyrupThinker) [inline docs/contributing.md:13]: It should be [Description](Link)

Comment (ry) [reply docs/contributing.md]: Fix thanks

**Comment by shortdiv** [comment]
This is a great! Thanks!

**Comment by Zambonifofex** [comment]
I think it might be a good idea to ensure people know that the Deno team is different from the Rust team. As it is, I can see people getting confused about it, and thinking they should click the “email the moderation team” on the Rust webpage for issues in the Deno community.

**Comment by boneskull** [comment]
Indeed, reporting CoC violations for Deno to the Rust moderation team would be ineffective.

**Comment by ry** [comment]
@Zambonifofex I think this will be sufficient.

**Comment by AdrieanKhisbe** [comment]
Suggestion: It might be useful to add a _Contributing_ link to the list of links in the [README](./README.md)

_(note: I could do this if asked for)_

**Pull request merged**